core/templates/dev/head/pages/dashboard/dashboard.html

```diff
@@ -123,6 +125,27 @@ <h1 class="stat-value"><[dashboardStats.total_open_feedback]></h1>
         </md-card>
       </div>
 
+      <div ng-if="explorationsList.length > 0" class="oppia-dashboard-tab-panel" style="padding-top: 13px;">
+        <span class="oppia-dashboard-total-elements">
+          You have a total of <[explorationsList.length]> explorations.
```

Comment:
I think we could omit these lines for explorations and collections. But we need this line for subscribers. I would make it translatable and add a condition for a single subscriber.